lib/awful/widget/calendar_popup.lua

```diff
@@ -246,9 +246,12 @@ end
 --
 -- @param widget Widget to attach the calendar
 -- @tparam[opt="tr"] string position Two characters string defining the position on the screen
+-- @tparam[opt={}] table args Additional options
+-- @tparam[opt=false] bool args.on_hover Show popup while mouse hover
```

Comment:
now i think i've got your point, we'll need to introduce additional state like `was_clicked` in order to handle it in `mouse_over` signal like `if not was_clicked then w.visible=false end`

i will update the PR tomorrow